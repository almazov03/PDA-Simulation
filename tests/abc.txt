STARTING_NON_TERMINAL=$S$
$S$ = { $S$ "a" }
$S$ = { $S$ "b" }
$S$ = { $S$ "c" }
$S$ = { EPS }