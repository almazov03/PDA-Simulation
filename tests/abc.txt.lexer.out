LexToken(START,'S',1,0)
LexToken(NON_TERMINAL,'S',2,27)
LexToken(EQUAL,'=',2,31)
LexToken(LBR,'{',2,33)
LexToken(TERMINAL,'a',2,35)
LexToken(NON_TERMINAL,'S',2,39)
LexToken(RBR,'}',2,43)
LexToken(NON_TERMINAL,'S',3,46)
LexToken(EQUAL,'=',3,50)
LexToken(LBR,'{',3,52)
LexToken(TERMINAL,'b',3,54)
LexToken(NON_TERMINAL,'S',3,58)
LexToken(RBR,'}',3,62)
LexToken(NON_TERMINAL,'S',4,65)
LexToken(EQUAL,'=',4,69)
LexToken(LBR,'{',4,71)
LexToken(TERMINAL,'c',4,73)
LexToken(NON_TERMINAL,'S',4,77)
LexToken(RBR,'}',4,81)
LexToken(NON_TERMINAL,'S',5,84)
LexToken(EQUAL,'=',5,88)
LexToken(LBR,'{',5,90)
LexToken(EMPTY,'EPS',5,92)
LexToken(RBR,'}',5,96)
